meinberlin/apps/bplan/models.py

```diff
@@ -1,30 +1,85 @@
+import sentry_sdk
 from django.contrib.auth.models import AnonymousUser
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from adhocracy4.administrative_districts.models import AdministrativeDistrict
 from adhocracy4.models.base import TimeStampedModel
 from adhocracy4.modules import models as module_models
+from adhocracy4.projects.models import Topic
 from meinberlin.apps.extprojects.models import ExternalProject
 
 
 class Bplan(ExternalProject):
     identifier = models.CharField(
         verbose_name=_("Identifier"),
         help_text=_(
-            "The identifier has to be identic to the identifier "
+            "The identifier has to be identical to the identifier "
             "in the FIS-Broker, so that district and location are "
             "added automatically."
         ),
         blank=True,
         max_length=120,
     )
+    is_diplan = models.BooleanField(default=False)
     office_worker_email = models.EmailField(
         verbose_name=_("Office worker email"),
         blank=True,
     )
 
+    def save(self, update_fields=None, *args, **kwargs):
+        if self.identifier:
+            district = self._get_district_for_identifier()
+            self.administrative_district = district
+            if update_fields:
+                update_fields.append("administrative_district")
```

Comment:
update_fields is a dict. Need to use the union as we did for other custom save() methods, see e.g [here](https://github.com/liqd/a4-meinberlin/blob/b64cad3c8cb2a32174a19ceee21c98a5660df95b/meinberlin/apps/ideas/models.py#L171)

Reply:
oh, good catch, thanks!